util/src/test/java/util/NumericalTest.java

```diff
@@ -30,7 +30,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-package org.cbioportal.genmone_nexus.annotation.util;
+package util;
```

Comment:
Seems like a minor refactoring issue, might be better to keep this class in the `org.cbioportal.genmone_nexus.util` package instead of just `util`